gov_uk_dashboards/lib/dap/get_dataframe_from_cds.py

```diff
@@ -19,6 +21,12 @@ def get_data_from_cds_or_fallback_to_csv(
     Returns:
         pd.DataFrame
     """
+    if (
+        "DATA_FOLDER_LOCATION" in os.environ
```

Comment:
Should data_folder_location be passed in as an argument?

Reply:
It should get set as an environment variable by a bit of code in run.py:
![image](https://user-images.githubusercontent.com/105639467/205868157-28d1d759-7efe-4be8-b1ac-0c15d1fba91d.png)
so shouldn't need passing in

Reply:
Thanks for explaining - this looks really good.